packages/clerk-js/src/ui/elements/Menu.tsx

```diff
@@ -17,24 +19,28 @@ type MenuState = {
   elementId?: MenuId;
 };
 
-const [MenuStateCtx, useMenuState] = createContextAndHook<MenuState>('MenuState');
+export const [MenuStateCtx, useMenuState] = createContextAndHook<MenuState>('MenuState');
 
-type MenuProps = PropsWithChildren<Record<never, never>> & { elementId?: MenuId };
+type MenuProps = PropsWithChildren<Record<never, never>> & {
+  elementId?: any;
```

Comment:
Why do we use `any` here instead of `MenuId` ?

Reply:
This happened because the trigger of `<ProfileSectionActionMenu/>` needs to have the same descriptor as the `<ProfileSectionButton/>` which is `descriptors.profileSectionPrimaryButton` so we don't have any breaking changes for customers that already using that descriptor, the change of the type `elementId` now is not only `MenuId` so I changed it to accept any value, I wanted to have this as a string because that is what are we passing but Typescript didn't liked that.

Reply:
It's better to align the types. You can do `type MenuId = 'invitation' | 'member' | ProfileSectionId`